CTFd/auth.py

```diff
@@ -503,6 +504,18 @@ def oauth_redirect():
 
                 team = Teams.query.filter_by(oauth_id=team_id).first()
                 if team is None:
+                    num_teams_limit = int(get_config("num_teams", default=0))
+                    num_teams = Teams.query.filter_by(
+                        banned=False, hidden=False
+                    ).count()
+                    if 0 < num_teams_limit <= num_teams:
```

Comment:
Why `0 <`? Couldn't this be:

```suggestion
                    if num_teams_limit and num_teams >= num_teams_limit:
```

Reply:
IIRC I started with something similar to the suggested code but then one of the CI linters wanted me to change it to what's in the commit. I'll try with this suggested code and see if it will accept it.